src/Workspaces/Core/Portable/Utilities/ForegroundThreadDataKind.cs

```diff
@@ -19,14 +19,6 @@ internal enum ForegroundThreadDataKind
 
     internal static class ForegroundThreadDataInfo
     {
-        private static readonly ForegroundThreadDataKind s_fallbackForegroundThreadDataKind;
-        private static ForegroundThreadDataKind? s_currentForegroundThreadDataKind;
-
-        static ForegroundThreadDataInfo()
-        {
-            s_fallbackForegroundThreadDataKind = CreateDefault(Unknown);
-        }
-
         internal static ForegroundThreadDataKind CreateDefault(ForegroundThreadDataKind defaultKind)
```

Comment:
Most surprised we're not burning this in a fire. Can we?

Reply:
➡️ Duplicate of one of the existing follow-up issues.